src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.h

```diff
@@ -1217,71 +1110,73 @@ class ArrayMeshStreamer :
 #endif
 
     if (deliverInline && destinationPe == this->myIndex_) {
-      ArrayDataItem<dtype, itype>
-        packedDataItem(arrayIndex, this->myIndex_, dataItem);
-      localDeliver(packedDataItem);
+      size_t sz = PUP::size(const_cast<dtype&>(dataItem));
+      char* data = new char[sz];
+      PUP::toMemBuf(const_cast<dtype&>(dataItem),data, sz);
```

Comment:
This looks to be a copy that can be avoided if we know the method data is an array of POD type, right?

Reply:
The new PUPbytes implementation resolves this